.travis.yml

```diff
@@ -8,9 +8,11 @@ before_install:
   - scripts/travis_es.sh
   - /tmp/elasticsearch/elasticsearch-1.3.7/bin/elasticsearch -d -D es.path.data=/tmp -D es.gateway.type=none -D es.index.store.type=memory -D es.discovery.zen.ping.multicast.enabled=false
 install:
+  - pip install flake8
```

Comment:
Agreed, we should probably add the `flake8 . --exclude=./docs/conf.py` command to the Makefile also so that you can easily check this locally.